debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -151,6 +151,7 @@ public void execute(ChangeEventSourceContext context, OraclePartition partition,
                     }
 
                     initializeRedoLogsForMining(jdbcConnection, false, startScn);
+                    long startTimeMillis = System.currentTimeMillis();
```

Comment:
@Naros We have `io.debezium.util.Stopwatch` - maybe it would be better to use it?